cpacs_gen_input/cpacs_schema.xsd

```diff
@@ -257,7 +257,11 @@ marko.alder@dlr.de
                                             <ddue:listItem>x-y-plane</ddue:listItem>
                                             <ddue:listItem>x-z-plane</ddue:listItem>
                                             <ddue:listItem>y-z-plane</ddue:listItem>
+                                            <ddue:listItem>none</ddue:listItem>
+                                            <ddue:listItem>inherit</ddue:listItem>
                                         </ddue:list>
+                                        By default, the symmetric of a component is inherited from its parent. If e.g. a wing is symmetricly defined, a pylon will also have a symmetry by default.
```

Comment:
Two typos here: "By default, the symmetry of a ..." and "... a wing is symmetrically defined, ..."

Reply:
Resolved in 
2ac0a1d